### DISCUSSION TITLE
Can I ensure the home link includes .index.html?

### DISCUSSION THREAD

**Discussion by aaronkyle** [observablehq/framework#1779] [Q&A] [answered]
This relates to  https://github.com/observablehq/framework/discussions/1680, which is closed as 'solved' so I am re-opening a similar question--hope that's OK.

I would like to ensure that my project uses verbose URLs.  In the [documentation on URLs](https://observablehq.com/framework/config#clean-urls), it specifies: "Regardless of this setting, a link to an index page will drop the implied index.html; for example foo/index.html will be formatted as foo/."  I have disabled clean URLs and so all the page navigation works as I wish, except for the home button on the site nav.  Can I manually specify this as a full path somehow?

**Comment by mbostock** [comment, marked as answer]
No, as documented, this is not currently supported. I’m curious why you need it?

**Comment by aaronkyle** [comment]
thanks @mbostock  - I am storing Framework projects on AWS S3.  If I don't publish a bucket as a static site, then I can't implement redirect.  If I do publish it as a static site, I have to use a different URL path.  The difference is
https://s3.us-east-1.amazonaws.com/data.exchange/example/dist/index.html
vs.
http://data.exchange.s3-website-us-east-1.amazonaws.com/example/dist/index.html

... it's not a big deal and I can use either; I was just sharing from the general bucket URL scheme and didn't initially realize the home button wasn't working, and now I have to change over some links and issue updates on the URL to folks (was hoping I could leave as is and silently fix).

**Comment by Fil** [comment]
duplicate of https://github.com/observablehq/framework/discussions/1680

**Reply by mbostock** [reply]
We should file a feature request for this…

**Comment by aaronkyle** [comment]
Raised here (feel free to edit if I didn't articulate well): https://github.com/observablehq/framework/issues/1782
